### ISSUE TITLE
Disable/exclude metrics buckets

### ISSUE THREAD

**Issue by gilliardmacedo** [labbsr0x/big-brother#49]
From https://github.com/labbsr0x/servlet-monitor/issues/31

Only more advanced users of Prometheus metrics really uses buckets/histograms. By removing metrics buckets, we will reduce in over 60% the amount of processing and byte output in our metrics endpoint with no loss in quality.

Currently we have something like this for each metrics/label combination:

```
request_seconds_bucket 1
request_seconds_bucket 2
request_seconds_bucket 3
request_seconds_bucket 4
request_seconds_bucket +inf
request_seconds_sum
request_seconds_count
response_size_bytes
```

By removing buckets, we would reduce to something like

```
request_seconds_sum
request_seconds_count
response_size_bytes
```

**Proposal**

- Buckets metrics made **optional** for request_seconds and dependency_request_seconds
- Disable buckets output by default (will only generate _sum and _count metrics for each request)
- Only enable buckets if configured explicitly using existing param "buckets" with values like "0.1,0.3,2,10" for example

**Comment by flaviostutz** [comment]
What do you think, guys? @abilioesteves

**Comment by eabili0** [comment]
@flaviostutz @gilliardmacedo makes sense to me. Sorry for such a late response (not really having time to open source as of now)
